.circleci/config.yml

```diff
@@ -54,20 +54,49 @@ commands:
           key: << parameters.python >>-creme_crm-{{ checksum "<< pipeline.parameters.source_path >>/setup.cfg" }}
           paths: "~/venv"
 
+  install-node-env:
+    steps:
+      - restore_cache:
+          key: node-dependencies-{{ checksum "<< pipeline.parameters.source_path >>/package-lock.json" }}
+
+      - run: cp << pipeline.parameters.source_path >>/package*.json << pipeline.parameters.instance_directory >>
+      - run:
+          name: Installing NodeJS
+          working_directory: << pipeline.parameters.instance_directory >>
+          command: npm install && npm run eslint-install
+          environment:
+            ESLINT_CREME_PLUGINS: "<< pipeline.parameters.source_path >>/creme/static/utils/eslint/"
+      - run:
+          working_directory: << pipeline.parameters.instance_directory >>
+          command: npm list
+
+      - save_cache:
+          key: node-dependencies-{{ checksum "<< pipeline.parameters.source_path >>/package-lock.json" }}
+          paths:
+            - << pipeline.parameters.instance_directory >>/node_modules
+
   create-creme-project:
     description: Create Creme project
-    parameters:
-      local_settings:
-        type: string
-        default: 'sqlite3_settings'
     steps:
       - run: mkdir << pipeline.parameters.instance_directory >>
       - run: touch << pipeline.parameters.instance_directory >>/__init__.py
       - run: mkdir << pipeline.parameters.instance_directory >>/media
       - run: mkdir << pipeline.parameters.instance_directory >>/media/static
-      # Not really useful because unit tests override with a sub-folder in /tmp/
```

Comment:
This comment is still relevant (see creme.creme_core.utils.test.CremeDiscoverRunner.setup_test_environment()).